curriculum/challenges/english/10-coding-interview-prep/rosetta-code/state-name-puzzle.md

```diff
@@ -15,7 +15,7 @@ dashedName: state-name-puzzle
 
 Write a function to solve the challenge for a given array of actual U.S. state names, and for fictional state names.
 
-The function should return an array. Each element should be an object in this form: `{from:[],to:[]}`. The `from` array should contain the original names and the `to` array should contain the resultant names.
+The function should return an array. Each element should be an object in this form: `{"from":[],"to":[]}`. The `"from"` array should contain the original names and the `"to"` array should contain the resultant names.
```

Comment:
here it should be the same thing as with `gallery`

Reply:
as in I mean the quotes should be removed only from the second sentence, not from the object